app/src/main/res/values/strings.xml

```diff
@@ -25,19 +25,19 @@
     <string name="accept">Accept</string>
     <string name="reject">Reject</string>
     <string name="stop">Stop</string>
-    <string name="sending">Sending</string>
+    <string name="sending">Sending…</string>
     <string name="cancel_sending">Cancel Sending</string>
     <string name="receiving">Receiving</string>
-    <string name="cancel_receiving">Cancel Receiving</string>
+    <string name="cancel_receiving">Cancel Reception</string>
```

Comment:
I think the original is better. This change will require a lot more changes because I don't call any other thing a reception in the app.

Reply:
In my head reception/recipient is the way to go.